### ISSUE TITLE
Status bar

### ISSUE THREAD

**Issue by kmaustral** [heliostatic/completed-task-display#2]
Thanks for this very useful plugin. 

Is it possible to have an option to remove the message in the status bar? Mine is getting very crowded and I don't really need this message. 

Much appreciated.

**Comment by heliostatic** [comment]
Agreed. I need to add a preference pane for optional display of status. I'll try to update this week.
